http/headers/cross-origin-resource-policy.json

```diff
@@ -42,7 +44,8 @@
               "version_added": false
             },
             "webview_android": {
-              "version_added": "73"
+              "version_added": "73",
+              "notes": "Until version 75, <a href='https://crbug.com/952834'>bug 952834</a> caused downloads to fail"
```

Comment:
```suggestion
              "notes": "Until version 75, downloads for files with this header would fail in WebView.  See <a href='https://crbug.com/952834'>bug 952834</a>."
```